src/MiNET/MiNET/Blocks/Block.cs

```diff
@@ -264,7 +153,9 @@ public virtual void BreakBlock(Level world, BlockFace face, bool silent = false)
 			}
 
 			UpdateBlocks(world);
-			world.BroadcastSound(Coordinates, LevelSoundEventType.BreakBlock, Id);
+
+			// not necessary?
+			//world.BroadcastSound(Coordinates, LevelSoundEventType.BreakBlock, GetRuntimeId());
```

Comment:
It's to make sure other players also hear block break

Reply:
DestroyBlockParticle spawn both sound and particle for all. (Minecraft feature)

need to remove this
// not necessary?
//world.BroadcastSound(Coordinates, LevelSoundEventType.BreakBlock, GetRuntimeId());